edalize/templates/vivado/vivado-run.tcl.j2

```diff
@@ -4,6 +4,8 @@ set_property STEPS.WRITE_BITSTREAM.ARGS.BIN_FILE true [get_runs impl_1]
 # Vivado will raise an error if impl_1 is launched when it is already done. So
 # check the progress first and only launch if its not complete.
 if { [get_property PROGRESS [get_runs impl_1]] != "100%"} {
+  launch_runs synth_1{{ jobs }}
```

Comment:
Can you add a comment why you launch the jobs separately? This information gets lost easily when doing the next refactoring.